packages/params/src/bannedContracts.ts

```diff
@@ -0,0 +1,22 @@
+import { NetworkName } from "types/lib/networks";
+
+export const BannedContracts: {
+  [name in NetworkName]?: string[];
+} = {
+  // https://doc.aurora.dev/evm/precompiles/#random-seed
+  aurora: ["0xc104f4840573bed437190daf5d2898c2bdf928ac"], // randomseed
+  auroraTest: ["0xc104f4840573bed437190daf5d2898c2bdf928ac"], // randomseed
+
+  // https://docs.moonbeam.network/builders/pallets-precompiles/precompiles/randomness/
+  moonbeam: ["0x0000000000000000000000000000000000000809"], // randomness
+  moonbase: ["0x0000000000000000000000000000000000000809"], // randomness
+
+  arbitrum: [
```

Comment:
Wont these be applicable for Optimism as well?

Reply:
Here are the optimism predeploys: https://github.com/ethereum-optimism/optimism/blob/develop/specs/predeploys.md

Reply:
added